main/ejml-dsparse/test/org/ejml/sparse/csc/mult/TestMatrixVectorMultWithSemiRing_DSCC.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2009-2020, Peter Abeles. All Rights Reserved.
+ *
+ * This file is part of Efficient Java Matrix Library (EJML).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ejml.sparse.csc.mult;
+
+import org.ejml.data.DMatrixSparseCSC;
+import org.ejml.ops.DSemiRing;
+import org.ejml.ops.DSemiRings;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TestMatrixVectorMultWithSemiRing_DSCC {
+    DMatrixSparseCSC inputMatrix;
+
+    @BeforeEach
+    public void setUp() {
+        // based on example in http://mit.bme.hu/~szarnyas/grb/graphblas-introduction.pdf
+        inputMatrix = new DMatrixSparseCSC(7, 7, 12);
+        inputMatrix.set(0, 1, 1);
+        inputMatrix.set(0, 3, 1);
+        inputMatrix.set(1, 4, 1);
+        inputMatrix.set(1, 6, 1);
+        inputMatrix.set(2, 5, 1);
+        inputMatrix.set(3, 0, 0.2);
+        inputMatrix.set(3, 2, 0.4);
+        inputMatrix.set(4, 5, 1);
+        inputMatrix.set(5, 2, 0.5);
+        inputMatrix.set(6, 2, 1);
+        inputMatrix.set(6, 3, 1);
+        inputMatrix.set(6, 4, 1);
+
+    }
+
+    @ParameterizedTest(name = "{0}")
+    @MethodSource("vectorMatrixMultSources")
+    public void mult_v_A(String desc, DSemiRing semiRing, double[] expected) {
+        // graphblas == following outgoing edges of source nodes
+        double[] v = new double[7];
+        Arrays.fill(v, semiRing.add.id);
+        v[3] = 0.5;
+        v[5] = 0.6;
+
+        System.out.println("input vector = " + Arrays.toString(v));
+
+        //input.print();
```

Comment:
zap/delete commented out debugging code

Reply:
Sure thing. I will also remove the prints in the tests